ci-operator/step-registry/gcp/provision/bastionhost/gcp-provision-bastionhost-commands.sh

```diff
@@ -195,3 +225,32 @@ rm -rf "${workdir}"
 
 echo "Sleeping 5 mins, make sure that the bastion host is fully started."
 sleep 300
+
+if [[ -f "${SHARED_DIR}/gcp_custom_endpoint" ]]; then
+  gcp_custom_endpoint=$(< "${SHARED_DIR}/gcp_custom_endpoint")
+
+  echo "$(date -u --rfc-3339=seconds) - Ensure GCP custom endpoint '${gcp_custom_endpoint}' is accessible..."
+
+  test_fqdn="compute-${gcp_custom_endpoint}.p.googleapis.com"
+  test_cmd="dig ${test_fqdn} | grep -q 'ANSWER: 1, '"
```

Comment:
Suggest to use `dig +short ${test_fqdn}` to judge its return value for easy reading.